src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public void setColor(String color) {
+        this.color = color;
+    }

```

Comment:
It will be more convenient to create a constructor for this class, which will let you create and fulfill an instance of this class using only 1 statement instead of 3.

Ball ball = new Ball();
ball.setNumber(smth);
ball.setColor(smth);

vs 

Ball ball = new Ball(smth, smth);